### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by ZIODIOZIO** [johngodley/redirection#3150]
```
Plugin: 5.1.3 4.2
WordPress: 5.8.2 (single)
PHP: 7.4.16
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.45 Safari/537.36
JavaScript: https://www.space-hosting.net/wp-content/plugins/redirection/redirection.js?ver=5.1.3-81f455fc2a8faec560fb955b29f9a717
REST API: https://space-hosting.net/wp-json/
Query: ?page=redirection.php&sub=options

Error: Failed to fetch
```

**Comment by johngodley** [comment]
You contacted me via three different methods. This creates 3 times as much work. Please use only one method.

As described in the error message your site URLs are not consistent. You should change them so they are the same.

**Comment by ZIODIOZIO** [comment]
Dear sorry for contacting you several times (I saw that you didn't reply to anyone on github so I thought about using the form but it didn't send me any confirmation email and so I contacted you via email).

From what you say it seems that my site has a different url (it makes me think of the type http: // mydomain https: // maildomain www.mydomain) but I can't understand where you see this error and where I should make of changes.
In wordpress settings?
